src/application/services/useEditor.ts

```diff
@@ -43,51 +44,57 @@ interface UseEditorParams {
  *
  * @param params - Editor.js params
  */
-export function useEditor({ id, content, isReadOnly, onChange }: UseEditorParams): void {
+export const useEditor = createSharedComposable(({ id, content, isReadOnly, onChange }: UseEditorParams): void => {
   /**
    * Editor instance
    */
   let editor: Editor | undefined;
 
+  /**
+   * Downloaded tools data
+   */
+  const downloadedTools = shallowRef<DownloadedTools>({})
```

Comment:
should it be reactive? maybe just `Set()`?

Reply:
if it's not reactive, I'm not sure, that `createSharedComposable` will work

Reply:
you can try